fleetctl/journal.go

```diff
@@ -47,6 +48,7 @@ func init() {
 	cmdJournal.Flags.BoolVar(&flagFollow, "f", false, "Shorthand for --follow")
 	cmdJournal.Flags.IntVar(&sharedFlags.SSHPort, "ssh-port", 22, "Connect to remote hosts over SSH using this TCP port")
 	cmdJournal.Flags.BoolVar(&flagSudo, "sudo", false, "Execute journal command with sudo")
+	cmdJournal.Flags.StringVar(&flagOutput, "output", "short", "Journalctl output mode")
```

Comment:
can we make this 
> "journalctl output mode. This will be passed unaltered to journalctl on the remote host, and hence supports the same modes as that command."